project/CodeCoverageSettings.scala

```diff
@@ -16,7 +16,7 @@ object CodeCoverageSettings {
 
   val settings: Seq[Setting[_]] = Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 100,
+    ScoverageKeys.coverageMinimumStmtTotal := 99,
```

Comment:
Looks like the only thing keeping it from 100% is the Enumerable reads stuff - is it worth putting a test in to bump coverage back to 100% or not?